### PR TITLE
Practitioner resource

### PR DESCRIPTION
**Pull request by pslawinski-htd**
Issue:  [Add FHIR Resource: Practitioner](https://app.zenhub.com/workspaces/patient-app-6140de7e31081800158d8ecf/issues/1uphealth/patient/8)

---

##### PR Checklist
- [x] Give this PR a meaningful title
- [x] Add a link to the related issue at the top of this description (above)
- [x] Connect this PR with the related issue via ZenHub with the button below this text box (or at the bottom of the page after the PR is created)
- [x] Ensure your branch is up to date with the target branch and resolve any conflicts
- [x] Answer the below questions to describe your PR for reviewers
- [x] Request at least two reviewers using the "Reviewers" section on the right, usually including at least one reviewer from your team
- [x] Notify the requested reviewers in the #code-review Slack channel once the PR is ready for review

---

## Why are these changes needed?

Practitioner resource styling has been revamped.

## What changed?

Zrzut ekranu 2021-10-27 o 16 34 12

## How are these changes tested?

Unit tests were updated to facilitate revamped resource. Storybook was used to test the resource visually.
Deployed storybook is available here: [link](http://fhir-react-lib-test-storybook.s3-website-us-east-1.amazonaws.com/branch/practitioner-resource/?path=/story/practitioner--default-visualization-dstu-2).

### CODE REVIEW

File: src/components/resources/Condition/Condition.js [hunk 1]
```diff
@@ -113,40 +105,52 @@ function Condition(props) {
...
+            icon={
+              <div className="pt-1 px-sm-1">
+                <HeaderIcon headerIcon={headerIcon}></HeaderIcon>
+              </div>
+            }
+            badgeStatus={clinicalStatus}
+            badge={<Badge data-testid="clinicalStatus">{clinicalStatus}</Badge>}
+            titleSegment={
+              <>
+                <Title>{codeText || ''}</Title>
```
Comment (sflejszman) [inline src/components/resources/Condition/Condition.js:121]: Can we set default value in destructuring?

File: src/components/resources/Procedure/Procedure.js [hunk 2]
```diff
@@ -50,45 +50,38 @@ const Procedure = props => {
...
-                          <MissingValue />
-                        )}
-                        {'   to   '}
-                        {performedPeriodEnd ? (
-                          <Date fhirData={performedPeriodEnd} />
-                        ) : (
-                          <MissingValue />
-                        )}
-                      </div>
+                    {'   to   '}
```
Comment (sflejszman) [inline src/components/resources/Procedure/Procedure.js:74]: Can we set space between elements using margin?

File: src/components/ui/index.js [hunk 3]
```diff
@@ -3,9 +3,18 @@ import './index.css';
 import React from 'react';
 
 export const Header = props => (
-  <div className="fhir-ui__Header d-flex w-100 align-items-start justify-content-between align-self-center">
-    {props.children}
-  </div>
+  <>
+    {// TODO comment why children
```
Comment (sflejszman) [inline src/components/ui/index.js:7]: To do?

File: src/components/ui/index.js [hunk 4]
```diff
@@ -3,9 +3,18 @@ import './index.css';
...
+    props.children ? (
```
Comment (sflejszman) [inline src/components/ui/index.js:8]: What do you think about use in this place:`props.children ||`

**Review by sflejszman — CHANGES_REQUESTED** [review]

**Review by sflejszman — APPROVED** [review]
🚀

**Review by liam1uphealth — APPROVED** [review]
lgtm

**Pull request merged**